ProductsAnalysisWeb/Controllers/MobileController.cs

```diff
@@ -19,25 +19,25 @@ public IActionResult Home()
         }
         public IActionResult Shop()
         {
-            List<Mobile> n = new List<Mobile>();
+            List<Mobile> mobilesList = new List<Mobile>();
```

Comment:
You can use 'var' keyword to define type if 'new' operator is used => because type can be seen just after 'new' statement
Also data type 'Mobile' better to rename to 'MobilePhone' as data entity must be noun or adj+noun...